src/System.IO.Packaging/src/System/IO/Packaging/ContentType.cs

```diff
@@ -688,7 +609,7 @@ private void EnsureParameterDictionary()
            '`' /*96*/, '|' /*124*/, '~'  /*126*/,
          };
 
-        private static readonly char[] s_forwardSlashSeparator = { '/' };
+        internal static readonly char[] s_forwardSlashSeparator = { '/' };
```

Comment:
I think we can do better than this. From what I see in this namespace, some classes use PackUriHelper.ForwardSlashChar (like ZipPackage.cs)

I was seeing it as we make ForwardSlashChar const and then add ForwardSlashCharSeparator as readonly char[] and then change all similar usages to use PackUriHelper's values.